marklogic-data-hub-central/ui/src/HCTheme.scss

```diff
@@ -150,4 +150,8 @@ $input-border-radius: 2px;
 
 .btn-sm, .btn-group-sm > .btn {
     padding: 0.05rem 0.45rem;
-  }
+}
+
+.cursor-pointer{
```

Comment:
Please add a space between the class name and the bracket here

Reply:
@mrodrigu-marklogic done